.github/workflows/combine-prs.yml

```diff
@@ -0,0 +1,151 @@
+name: 'Combine PRs'
```

Comment:
Would it make sense to document a little bit what this workflow does? In case any developer arrives here with doubts about the process and can avoid reading all the code to understand it. 

At least topics like which is the result of executing this workflow and which conditions are applied (only dependabot, the label to skip it, etc.) I'd find it useful, but I see no other workflows are documented so let's see your opinion :)